torchvision/transforms/v2/functional/_utils.py

```diff
@@ -169,3 +169,11 @@ def _is_cvcuda_available():
         return True
     except ImportError:
         return False
+
+
+def is_cvcuda_tensor(inpt: Any) -> bool:
```

Comment:
Let's communicate that this is private:


```suggestion
def _is_cvcuda_tensor(inpt: Any) -> bool:
```